segment_batching_script.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sct_run_batch -jobs 6 -path-data "/PATH/TO/dataset" \
+	-subject-prefix "sub-" -exclude-list sub-107 sub-125 -script segment_sc_discs_deepseg.sh \
+	-path-output "/PATH/TO/dataset/derivatives/labels"
```

Comment:
### 5. Providing steps to test/reproduce 

Currently, the README.md does a good job at providing general steps for people bringing their own datasets. But, as a reviewer without a dataset, I don't really know what I need to do in order to follow these steps myself.

I notice in commit 9b4d4c6fda6c4d68653fc6d6efe1416e17f64b3d there was reference to a dataset called `./philidelphia-pediatric` in the `-path-data` argument to `sct_run_batch`. I was curious to try to download this dataset and perhaps follow the steps in the README.md. But, the PR description doesn't contain any guidance for downloading and running the steps, so I'm not sure which repo/server to get the data from.